examples/hitl/rearrange_v2/rearrange_v2.py

```diff
@@ -349,31 +358,29 @@ def _update_grasping_and_set_act_hints(self, user_index: int):
             reach_pos=None,
         )
 
-    def _get_gui_controlled_agent_index(self, user_index) -> int:
+    def _get_gui_controlled_agent_index(self, user_index):
         return self._gui_agent_controllers[user_index]._agent_idx
 
     def _get_controls_text(self, user_index: int):
-        if self._paused:
-            return "Session ended."
-
-        if not self._user_data[user_index].show_gui_text:
-            return ""
-
         controls_str: str = ""
-        controls_str += "H: Toggle help\n"
-        controls_str += "Look: Middle click (drag), I, K\n"
-        controls_str += "Walk: W, S\n"
-        controls_str += "Turn: A, D\n"
-        controls_str += "Finish episode: Zero (0)\n"
-        controls_str += "Open/close: Double-click\n"
-        controls_str += "Pick object: Double-click\n"
-        controls_str += "Place object: Right click (hold)\n"
+        if self._user_data[user_index].show_gui_text:
+            controls_str += "H: Toggle help\n"
+            controls_str += "Look: Middle click (drag), I, K\n"
+            controls_str += "Walk: W, S\n"
+            controls_str += "Turn: A, D\n"
+            controls_str += "Finish episode: Zero (0)\n"
+            controls_str += "Open/close: Double-click\n"
+            controls_str += "Pick object: Double-click\n"
+            controls_str += "Place object: Right click (hold)\n"
```

Comment:
Is there an option to see current-instruction?

Reply:
Yes, it's currently displayed via the status message.